lua/dropbar/sources/path.lua

```diff
@@ -148,7 +165,7 @@ local function convert(path, buf, win)
   return bar.dropbar_symbol_t:new(setmetatable({
     buf = buf,
     win = win,
-    name = vim.fs.basename(path),
+    name = (path == '/') and path or vim.fs.basename(path),
```

Comment:
On posix systems this handles adding the slash after the folder icon at the root of the breadcrumbs, otherwise it is just the folder icon with no text